file

```diff
@@ -0,0 +1,1425 @@
+Active Internet connections (w/o servers)
```

Comment:
@Stanislaw-bnk this definitely should not be here.

Reply:
Don't know where it came from. I have deleted this file.